cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -234,8 +237,15 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         return ret
     }
 
+    /**
+     * Translates a Python [Assign](https://docs.python.org/3/library/ast.html#ast.Assign] into a
+     * [Statement].
+     */
     private fun handleAssign(node: Python.AST.Assign): Statement {
```

Comment:
we can also return an assign expression here now. I think we previously returned a statement here because of the problem expressions.